### DISCUSSION TITLE
Get kiosk with Chrome running?

### DISCUSSION THREAD

**Discussion by quadcom** [labwc/labwc#2301] [Q&A]
I switched from Wayland to labwc on my little Pi5-Bookworm. Bu of course that killed my kiosk that I had running. 

I tried going through the docs and figuring it out but not being a seasoned Linux guru I'm a bit lost.

Can anyone walk me through getting this running again?

Thanks in advance!

**Comment by Consolatis** [comment]
I suggest using [cage](https://github.com/cage-kiosk/cage) for kiosk setups. Labwc is not targeted for that use-case and thus it might work but then has various pain points when actually using it in such an environment.

Another option is to simply switch back to wayfire via raspi config (if that is what you were using previously).

**Comment by johanmalm** [comment]
Do you mean that you were running `chromium --kiosk` before and that the kiosk mode now does not work? If so what happens? Is it just in normal browser mode?

**Comment by rastersoft** [comment]
In my case, what I did was to edit `/etc/xdg/labwc/autostart`, and commented everything except the `/usr/bin/kanshi &` command, and added my kiosk program *after* that command (I need it because the screen in my system is rotated, and that `kanshi` daemon is required to be running before launching any other program to update the rotation).

**Reply by genebean** [reply]
FWIW, I just appended the chromium command from the official kiosk tutorial to `/etc/xdg/labwc/autostart` and it worked fine for me. This was done tonight on a fresh install of 64bit Raspberry Pi OS.

**Comment by me21** [comment]
Also, if you're running a different application, you may have to add the following window rule to `/etc/xdg/labwc/rc.xml` file:
```
    <windowRule identifier="your_app_id">
        <action name="ToggleFullscreen"/>
    </windowRule>
```
